analyzer/src/main/kotlin/managers/utils/MavenDependencyHandler.kt

```diff
@@ -78,11 +84,23 @@ class MavenDependencyHandler(
     override fun linkageFor(dependency: DependencyNode): PackageLinkage =
         if (isLocalProject(dependency)) PackageLinkage.PROJECT_DYNAMIC else PackageLinkage.DYNAMIC
 
+    /**
+     * Create a [Package] representing [dependency] if possible, adding potential issues to [issues]. Inter-project
```

Comment:
Nits:
- Maybe "the" / "a" before "dependency"?
- Maybe try to avoid the "issues" word duplication, like "... recording any [issues]." (Or "tracking", "reporting".)